make_dashboards/templates/external/js/dashboard.js

```diff
@@ -263,9 +263,9 @@ function make_throughput_plot(){
         });
     }
     // Subtitle text
-    var bp_per_week = total_count / num_weeks;
-    var mbp_per_second = (bp_per_week / (60*60*24*7))/1000000;
-    var subtitle_text = 'Average per week: '+parseInt(bp_per_week/1000000000)+' Gbp ('+parseInt(mbp_per_second)+' million base pairs per second)';
+    var bp_per_day = total_count / (num_weeks * 7);
+    var genomes_per_minute = (bp_per_day / (24*60)) / 3236336281;
```

Comment:
Yeah, I tried to phrase it carefully - "human genome equivalents". I think 1x makes more sense - just because we sequence genomes thirty times over to do variant calling doesn't mean that everyone else does.